src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <div class="card__product_image">
+        <img
+          class="card__product_image_img"
+          src="./images/imac.jpeg"
+          alt="Моноблок APPLE A1419 iMac 27"
+        >
+      </div>
+      <div class="card__product_name">
+        <span>
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </span>
+      </div>
+      <div class="card__product_code">
+        <span>
+          Код товара: 195434
+        </span>
+      </div>
+      <div class="card__product_review">
+        <div class="card__review_stars">
+          <div class="card__review_stars_star"></div>
+          <div class="card__review_stars_star"></div>
+          <div class="card__review_stars_star"></div>
+          <div class="card__review_stars_star"></div>
+          <div class="card__review_stars_star"></div>
+        </div>
+        <div class="card__review_text">
+          <a
+            class="card__review_text_link"
+            href="#"
+            data-qa="hover"
+          >
+            Отзывов: 5
+          </a>
+        </div>
+      </div>
+      <div class="card__product_price">
+        <div class="card__product_price_text">
+          <span>
+            Цена:
+          </span>
+        </div>
+        <div class="card__product_price_value">
+          <span>
+            69 999 грн
+          </span>
+        </div>
+      </div>
+      <div class="card__product_button">
+        <a
+          data-qa="hover"
+          class="card__product_button_btn"
+          href="#"
+        >
+          Купить
+        </a>
+      </div>
```

Comment:
```suggestion
      <a
        data-qa="hover"
        class="card__product_button_btn"
        href="#"
      >
        Купить
      </a>
```
щоб тести пройшли, можна забрати цей div і стилізувати саму лінку як кнопку

Reply:
Прибрав обгортку div, але результат той самий. Не пофіксило :-(

Reply:
зробив. І навть підправив трішки файл reset.css, але результат той самий. Цікаво, що тест сприймає кнопку з :hover наче без :hover, тобто фон залишається синім, а не білим, як я прописав для :hover

Reply:
Ок. Це не критично. Хороша робота у будь-якому випадку. Молодець 👍

Reply:
Дякую! Йдемо далі! ++